packages/grafana-data/src/types/forms.ts

```diff
@@ -0,0 +1,3 @@
+export interface FormAPI extends Record<string, any> {
+  errors: Record<string, any>;
```

Comment:
Should FormAPI extend react-hook-form like in grafana-ui: 
`export type FormAPI<T extends FieldValues> = Omit<UseFormReturn<T>, 'trigger' | 'handleSubmit'> & {
  errors: FieldErrors<T>;
};
` to support advanced validation?